converter/src/main/java/gov/cms/qpp/conversion/validate/AciNumeratorDenominatorValidator.java

```diff
@@ -53,11 +39,11 @@ protected void internalValidateSingleNode(Node node) {
 	 */
 	private void validateChildren(Checker nodeChecker) {
 		nodeChecker
-				.singleValue(NO_MEASURE_NAME, "measureId")
-				.hasChildren(NO_CHILDREN)
-				.childMinimum(NO_DENOMINATOR, 1, TemplateId.ACI_DENOMINATOR)
-				.childMinimum(NO_NUMERATOR, 1, TemplateId.ACI_NUMERATOR)
-				.childMaximum(TOO_MANY_DENOMINATORS, 1, TemplateId.ACI_DENOMINATOR)
-				.childMaximum(TOO_MANY_NUMERATORS, 1, TemplateId.ACI_NUMERATOR);
+				.singleValue(ErrorCode.ACI_NUMERATOR_DENOMINATOR_MISSING_MEASURE_ID, "measureId")
+				.hasChildren(ErrorCode.ACI_NUMERATOR_DENOMINATOR_MISSING_CHILDREN)
+				.childMinimum(ErrorCode.ACI_NUMERATOR_DENOMINATOR_VALIDATOR_MISSING_DENOMINATOR_CHILD_NODE, 1, TemplateId.ACI_DENOMINATOR)
+				.childMinimum(ErrorCode.ACI_NUMERATOR_DENOMINATOR_VALIDATOR_MISSING_NUMERATOR_CHILD_NODE, 1, TemplateId.ACI_NUMERATOR)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 122 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)